src/benchq/compilation/pyliqtr_transpilation.py

```diff
@@ -56,12 +57,17 @@ def pyliqtr_transpile_to_clifford_t(
         raise ValueError("Please supply gate or circuit precision not both")
 
     cirq_circuit = export_circuit(CirqCircuit, orquestra_circuit)
-    compiled_cirq_circuit = clifford_plus_t_direct_transform(
-        cirq_circuit,
-        precision=gate_precision,
-        circuit_precision=circuit_precision,
-        use_random_decomp=False,
-        num_rotation_gates=n_rotation_gates,
-    )
+    with warnings.catch_warnings():
+        warnings.filterwarnings("ignore", category=RuntimeWarning)
+        # Pyliqtr throws "RuntimeWarning: H is not a rotation gate, cannot decompose.
+        # Set warn_if_not_decomposed=False to silence this warning." but the setting
+        # to silent warning is not exposed by pyliqtr.
```

Comment:
Did you ever figure out what happened here? I asked especially for that option and it didn't work once they made it available. 😆

Reply:
Oh my texteditor lied to me! There is actually such parameter and it seems to be working! Thanks for pointing that out

Reply:
Oh I was wrong afterall.
warn_if_not_decomposed is a parameter for this function, but it is not passed correctly into
determine_gate_precision

../venvs/benchq3.9_1/lib/python3.9/site-packages/pyLIQTR/gate_decomp/cirq_transforms.py:291: in clifford_plus_t_direct_transform
    gate_precision = determine_gate_precision(
../venvs/benchq3.9_1/lib/python3.9/site-packages/pyLIQTR/gate_decomp/cirq_transforms.py:467: in determine_gate_precision
    num_rotation_gates = get_num_rotation_gates(circuit)
../venvs/benchq3.9_1/lib/python3.9/site-packages/pyLIQTR/gate_decomp/cirq_transforms.py:485: in get_num_rotation_gates
    if parse_gate(op.gate) is not None:

Heres full stacktrace if we want to report it as a bug into pyliqtr

Reply:
The TLDR of my previous message: they added parameter that is not working properly as you mentioned :)

Reply:
Ok I'll have another chat with justin and we'll work it out.